Library/Formula/dependency-check.rb

```diff
@@ -25,6 +25,10 @@ def install
     (etc/"dependencycheck").mkpath
     corejar = libexec/"repo/org/owasp/dependency-check-core/#{version}/"\
       "dependency-check-core-#{version}.jar"
+    if File.exist? etc/"dependencycheck/dependencycheck.properties"
```

Comment:
Please use `etc.install` which handles the modified case and overwriting automatically, thanks.